README.md

```diff
@@ -1,64 +1,82 @@
-## About
+## Open Source Routing Machine
 
-The Open Source Routing Machine is a high performance routing engine written in C++11 designed to run on OpenStreetMap data.
+[![Travis](https://travis-ci.org/Project-OSRM/osrm-backend.png?branch=master)](https://travis-ci.org/Project-OSRM/osrm-backend)
```

Comment:
I have to say that some information regarding the systems used should be here. Before we had the table with linux/windows. I wouldn't have been able to tell if the builds are passing on mac, linux, windows based on the buttons alone before I worked here. So I feel having information here would benefit the readability.
What is your take? Is this clear from the buttons alone?